packages/translate/lib/__tests__/progress.test.ts

```diff
@@ -0,0 +1,38 @@
+import {PROGRESS_STATUS} from '../constants';
+import {translateProgress} from '../progress';
+
+describe('Interactive progress explanation.', () => {
+	// No need for deep testing because translateProgress with code will return the same result as translateStartupError
+	it('Should return translation if code is valid', () => {
+		const res = {
+			title: 'Cannot continue: Your subscription has expired',
+			description: 'Your subscription has expired, to continue using Suitest please [renew your subscription](https://the.suite.st/preferences/billing)',
+		};
+		expect(translateProgress({code: 'noActivePlan'})).toStrictEqual(res);
+		expect(translateProgress({code: 'noActivePlan', status: 'actionFailed'})).toStrictEqual(res);
+	});
+
+	describe('Translation result for', () => {
```

Comment:
describe('Interactive progress translations ') 
    it(`"Should translate "${status}" status`)
?